sos/plugins/ceph.py

```diff
@@ -50,23 +50,41 @@ def setup(self):
         ])
 
         self.add_cmd_output([
-            "ceph status",
-            "ceph health detail",
-            "ceph osd tree",
-            "ceph osd stat",
-            "ceph osd dump",
             "ceph mon stat",
-            "ceph mon dump",
-            "ceph df",
+            "ceph-disk list",
+            "ceph osd erasure-code-profile ls",
+            "ceph -v"
+        ])
+
+        ceph_cmds = [
+            "status",
+            "health detail",
+            "osd tree",
+            "osd stat",
+            "osd df tree",
+            "osd dump",
+            "osd df",
+            "mon dump",
+            "df",
+            "df detail",
+            "fs ls",
+            "fs dump",
+            "pg dump",
+            "-s"
+        ]
+
+        self.add_cmd_output([
+            "ceph %s" % s for s in ceph_cmds
+        ])
+
+        self.add_cmd_output([
+            "ceph %s --format json-pretty" % s for s in ceph_cmds
+        ], subdir="json_output")
+
+        self.add_cmd_output([
             "ceph report",
-            "ceph osd df tree",
-            "ceph fs dump --format json-pretty",
-            "ceph fs ls",
-            "ceph pg dump",
-            "ceph health detail --format json-pretty",
             "ceph osd crush show-tunables",
-            "ceph-disk list"
-        ])
+        ], subdir="json_output")
```

Comment:
@zerodayz these two commands shouldn't be under the 'json_output' subdir. Please keep them under the previous path.